app/assets/javascripts/models/beta/story.js

```diff
@@ -129,7 +130,7 @@ export const deleteStory = (storyId, projectId) =>
   httpService
     .delete(`/projects/${projectId}/stories/${storyId}`);
 
-export const updateStory = (story, newAttributes) => ({
+export const addNewAttributes = (story, newAttributes) => ({
```

Comment:
I believe this function does the same thing as [`extend`](https://underscorejs.org/#extend) from underscore.
Or you can replace `addNewAttributes` to `extend` or call `extend` into `addNewAttributes`. I think the second option is the easiest one.

```js
import { extend } from 'underscore'
...
export const addNewAttributes = extend;
```

Reply:
Nice! Thanks nolleto!